crates/voicevox_core/src/devices.rs

```diff
@@ -2,15 +2,39 @@ use serde::{Deserialize, Serialize};
 
 use super::*;
 
+/// このライブラリで利用可能なデバイスの情報。
+///
+/// あくまで本ライブラリが対応しているデバイスの情報であることに注意。GPUが使える環境ではなかったと
+/// しても`cuda`や`dml`は`true`を示す。
```

Comment:
これは明記しておかないと混乱を招くかと。

Reply:
たしかに。かなり重要な気がしました。

mayであることをちょっとわかりやすくしても良いかもとかちょっと思ったのでsuggestionをば。

```suggestion
/// このライブラリで利用可能なデバイスの情報。
///
/// あくまで本ライブラリが対応しているデバイスの情報であることに注意。GPUが使える環境ではなかったと
/// しても`cuda`や`dml`は`true`を示しうる。
```

Reply:
なるほど (C/Python側もあるので、commit suggestionは後で押します)